contrib/mpi-proxy-split/mpi-wrappers/mpi_request_wrappers.cpp

```diff
@@ -64,6 +64,7 @@ USER_DEFINED_WRAPPER(int, Test, (MPI_Request*) request,
     return MPI_SUCCESS;
   }
   DMTCP_PLUGIN_DISABLE_CKPT();
+  LOG_PRE_Test(status);
```

Comment:
Thanks for catching this missing macro.

Can we move this line one higher, to before `DMTCP_PLUGIN_DISABLE_CKPT();`?

It should be safe to allow checkpointing in the middle of this macro for logging.  DMTCP can safely save the open files associated with logging.  So, there's no reason to temporarily disable and delay checkpointing while we write to the log file.